nuxeo-platform-video-core/src/test/java/org/nuxeo/ecm/platform/video/extension/TestVideoImporterAndListeners.java

```diff
@@ -167,7 +152,13 @@ public void testVideoType() {
         assertEquals("testTitle", docModelResult.getPropertyValue("dc:title"));
         assertEquals("testUser", docModelResult.getPropertyValue("picture:credit"));
         assertEquals("testUid", docModelResult.getPropertyValue("uid:uid"));
-        assertEquals("0.0", docModelResult.getPropertyValue(DURATION_PROPERTY).toString());
+
+        docModelResult = session.getDocument(docModelResult.getRef());
+
+        // no video blob so expecting null/empty values for info, storyboard and conversions
+        assertNull(docModelResult.getPropertyValue(DURATION_PROPERTY));
+        assertTrue(((List) docModel.getPropertyValue("vid:storyboard")).isEmpty());
```

Comment:
Or you can do this, so that in case of failure there is more information:

    assertEquals(Collections.emptyList(), docModel.getPropertyValue("vid:storyboard"));

Reply:
:+1: